azure_arc_servers_jumpstart/azure_stack_hci/powershell/azstack_hci_vm_deploy.ps1

```diff
@@ -0,0 +1,137 @@
+#Powershell script to deploy VM in Azure Stack HCI and Arc enable it
+
+# Environment variables for the optional configurations
+$DHCPEnabled = 'Select $true if DHCP is enabled on your environment, $false if not'
+
+    #If $DHCPEnabled = $false, please fill the following variables. Otherwise atribbute the vale $null
+    $IPAddress = 'Provide the static IP to assign the to the VM'
+    $PrefixLenght = 'Provide the lenght of the subnet mask to assing to the VM'
+    $DefaultGateway = 'Provide the default gateway to assign to the VM'
+    $DNSServer = 'Provide the DNS Server to assign to the VM'
+
+$ServerClusterEnabled = 'Select $true if you have a server cluster created, $false if the not'
+    
+    #If $ServerClusterEnabled = $true,  please provide the path to the cluster storage in the format "<Disk Letter>:\Folder"
+    #If $ServerClusterEnabled = $false, please provide the path to a folder where the VM will be created in the format "<Disk Letter>:\Folder"
+    $vmdir =  "<Disk Letter>:\Folder"
```

Comment:
```suggestion
#Powershell script to deploy VM in Azure Stack HCI and Arc enable it

# Environment variables for the optional configurations
$DHCPEnabled = 'Select $true if DHCP is enabled on your environment, $false if not'

    #If $DHCPEnabled = $false, please fill the following variables. Otherwise, attribute the value $null
    $IPAddress = 'Provide the static IP to assign the to the VM'
    $PrefixLenght = 'Provide the length of the subnet mask to assign to the VM'
    $DefaultGateway = 'Provide the default gateway to assign to the VM'
    $DNSServer = 'Provide the DNS Server to assign to the VM'

$ServerClusterEnabled = 'Select $true if you have a server cluster created, $false if the not'
    
    #If $ServerClusterEnabled = $true,  please provide the path to the cluster storage in the format "<Disk Letter>:\Folder"
    #If $ServerClusterEnabled = $false, please provide the path to a folder where the VM will be created in the format "<Disk Letter>:\Folder"
    $vmdir =  "<Disk Letter>:\Folder"
```